src/test/java/org/junitpioneer/jupiter/ExpectedToFailExtensionTests.java

```diff
@@ -42,7 +41,7 @@ void abortsOnTestFailure() {
 				.whichAborted()
 				.withExceptionInstanceOf(TestAbortedException.class)
 				.hasMessage("Test marked as temporarily 'expected to fail' failed as expected")
-				.hasCause(new AssertionFailedError("failed"));
+				.hasCause(new AssertionError("failed"));
```

Comment:
Don't we throw `AssertionFailedError` now?

Reply:
We do, but this is not actually us - it's AssertJ (with the `fail()` method). You could argue that checking _why_ this test failed is not necessary?

Reply:
I see, then `AssertionError` is fine I guess (also we don't use `hasCauseExactlyInstanceOf`).